safety-net.php

```diff
@@ -29,6 +29,7 @@
 require_once __DIR__ . '/includes/utilities.php';
 require_once __DIR__ . '/includes/deactivate-plugins.php';
 require_once __DIR__ . '/includes/scrub-options.php';
+require_once __DIR__ . '/includes/disable-webhooks.php';
```

Comment:
You'll probably want to upgrade the version to `1.4.23` at the top of this file.

Reply:
Updated in 3a10b86